examples_utils/benchmarks/run_benchmarks.py

```diff
@@ -122,6 +124,14 @@ def run_and_monitor_progress(
     outs = [[], []]
     ipu_monitoring: List[str] = []
 
+    def kill_process(proc_pid):
```

Comment:
Can you add the type of the pid? int I think?

```suggestion
    def kill_process(proc_pid: int):
        """Kill a process by recursively killing it's child processes first"""
```